analyzer/codechecker_analyzer/analyzer_context.py

```diff
@@ -116,9 +119,24 @@ def __set_version(self):
             self.__package_git_tag = package_git_dirtytag
 
     def __populate_analyzers(self):
+        """ Set analyzer binaries for each registered analyzers. """
+        check_env = None
+        analyzer_from_path = is_analyzer_from_path()
+        if analyzer_from_path:
+            check_env = extend(self.path_env_extra,
```

Comment:
I think using the package name makes it a bit more readable `env.extend` and will not conflict with any local `extend` later.